app/src/main/java/com/example/myweatherapp/data/mapper/WeatherMapper.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.myweatherapp.data.mapper
+
+import com.example.myweatherapp.data.entity.Data
+import com.example.myweatherapp.data.entity.Temperature
+import com.example.myweatherapp.data.entity.Weather
+import com.example.myweatherapp.data.entity.WeatherDescription
+import com.example.myweatherapp.data.service.response.WeatherDataResponse
+import com.example.myweatherapp.data.service.response.WeatherFullDescriptionResponse
+import com.example.myweatherapp.data.service.response.WeatherResponse
+import com.example.myweatherapp.data.service.response.WeatherTemperatureResponse
+
+class WeatherMapper {
```

Comment:
mappers can be aplied as extension functions in a kotlin file, no need to wrap them in a class

Reply:
Done, thx for all your help!!